### PR TITLE
fix(isObservable): Fix throwing error when testing isObservable(null)

### PR DESCRIPTION
**Pull request by bbonnet**
**Description:**
Fix `isObservable` to test to make sure the test subject is a function or object before attempting to access properties on it.

**Related issue (if exists):**
https://github.com/ReactiveX/rxjs/issues/3687

### CODE REVIEW

**Comment by bbonnet** [comment]
Above gave suggestion gave me `AssertionError: expected null to be false` for the `isObservable(null)` case so added `!!obj` for coercion to boolean

File: src/internal/util/isObservable.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { Observable } from '../Observable';
 import { ObservableInput } from '../types';
+import { isObject  } from './isObject';
+import { isFunction } from './isFunction';
```
Comment (demenskiy) [inline src/internal/util/isObservable.ts]: Remove unused imports.

Comment (bbonnet) [reply src/internal/util/isObservable.ts]: Good catch, updated.

**Comment by benlesh** [comment]
thanks @bbonnet!

**Pull request merged**